com.salesforce.b2eclipse.jdt.ls/src/main/java/com/salesforce/b2eclipse/managers/BazelProjectImporter.java

```diff
@@ -67,20 +70,19 @@
 
     @Override
     public boolean applies(IProgressMonitor monitor) throws OperationCanceledException, CoreException {
-        B2EPreferncesManager preferencesManager = B2EPreferncesManager.getInstance();
-        if (preferencesManager != null && !preferencesManager.isImportBazelEnabled()) {
-            return false;
-        }
-        if (!rootFolder.exists() || !rootFolder.isDirectory()) {
-            return false;
-        }
-        File workspaceFile = new File(rootFolder, WORKSPACE_FILE_NAME);
-        if (workspaceFile.exists()) {
-            directories = Arrays.asList(Path.of(rootFolder.getPath(), new String[0]));
-        } else {
+        // Since the actual preferences come from the VSCode via LSP, it is not defined
```

Comment:
what about other B2E clients - i.e. Eclipse?

Reply:
It will still work well for others.